### PR TITLE
Add the Swan Lake 2201.2.0 announcement blog

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the Swan Lake 2201.2.0 announcement blog.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by dakshitha — APPROVED** [review]

File: _posts/2022-08-10-announcing-ballerina-2201.2.0-swan-lake-update-2.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+- You can now get the status of strands and strand groups during the execution of a Ballerina program, which can be used to troubleshoot runtime errors. The Ballerina runtime will emit the strand dump to the standard output stream in the text format if it receives a `SIGTRAP` signal (`SIGTRAP` is not available on Windows).
+- You can also register a function that will be called during a graceful shutdown. A call to `onGracefulStop` will result in one call to the handler function that was passed as an argument; the handler functions will be called after calling `gracefulStop` on all registered listeners in the reverse order of the corresponding calls to `onGracefulStop` (e.g., the function foo can be called during the graceful shutdown by registering it as follows: `runtime:onGracefulStop(foo);`).
+- In terms of improvements, when a type is defined referring to another type, it will now be passed to the runtime as a `BTypeReferenceType` instance. A few existing runtime APIs have been modified to return the `BTypeReferenceType` instances.
+
+The following standard library and extended library modules now come with new features and improvements:
```
Comment (anupama-pathirage) [inline _posts/2022-08-10-announcing-ballerina-2201.2.0-swan-lake-update-2.md:37]: Shouldn't we have a subtitle like `Improvements to the standard library` to be consistent with the previous subtitle of `Improvements to the language`?

File: _posts/2022-08-10-announcing-ballerina-2201.2.0-swan-lake-update-2.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## `rabbitmq` 
+
+- Added constraint validation support for payload binding
+
+## `nats`
+
+- Added constraint validation support for payload binding
+
+Here are the major new features and improvements you can find in the developer tools:
```
Comment (anupama-pathirage) [inline _posts/2022-08-10-announcing-ballerina-2201.2.0-swan-lake-update-2.md:105]: Same as above. Shouldn't we have a sub title saying `Improvements to the developer tools`?

File: _posts/2022-08-10-announcing-ballerina-2201.2.0-swan-lake-update-2.md (hunk 1 above)
Comment (praneesha) [reply _posts/2022-08-10-announcing-ballerina-2201.2.0-swan-lake-update-2.md:37]: Added.

File: _posts/2022-08-10-announcing-ballerina-2201.2.0-swan-lake-update-2.md (hunk 2 above)
Comment (praneesha) [reply _posts/2022-08-10-announcing-ballerina-2201.2.0-swan-lake-update-2.md:105]: Added.

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**